fcrepo-http-api/src/test/java/org/fcrepo/integration/http/api/FedoraLdpIT.java

```diff
@@ -3203,13 +3195,16 @@ public void testCreateResourceWithoutContentType() {
         assertEquals(CREATED.getStatusCode(), getStatus(new HttpPut(serverAddress + getRandomUniqueId())));
     }
 
+    /*
+     * TODO: This was originally expecting a 415 Unsupported Media Type, but because we assume text/turtle now it
```

Comment:
Is there actually anything "todo"?

Reply:
Sorry I just wanted to keep track of these little changes to the interactions. If going from 415 to 400 is fine, then I can remove the `TODO`.

Reply:
The change in response code here is probably ok.